misc/testerina/modules/testerina-core/src/main/ballerina/execute.bal

```diff
@@ -13,17 +13,20 @@
 // KIND, either express or implied.  See the License for the
 // specific language governing permissions and limitations
 // under the License.
-
 import ballerina/lang.'error as langError;
 
-boolean shouldSkip = false;
+isolated boolean shouldSkip = false;
 boolean shouldAfterSuiteSkip = false;
-int exitCode = 0;
+isolated int exitCode = 0;
+isolated final ConcurrentExecutionManager conMgr = new;
```

Comment:
Now that `ConcurrentExecutionManager` is `isolated`, this can be just a final variable, right?

Please recheck final vs isolated usage, wherever applicable. If we can work with just final, we should do that and avoid unnecessary locks and associated clones.